shared/utils/galaxy_ansible_role_meta_template.yml

```diff
@@ -0,0 +1,17 @@
+galaxy_info:
+  author: OpenSCAP
+  description: @DESCRIPTION@
+  company: Red Hat, Inc.
```

Comment:
we shouldn't do this in an upstream community project

Reply:
Humm, ok, so OpenSCAP again?